### PR TITLE
Fix sectionInset.bottom when footers are not in use

### LINKED ISSUE #110: Configuration without footers missing sectionInsetBottom
When you don't use footers the section inset bottom is missed out in the calculations for the nextY.

### PR DESCRIPTION
**Pull request by christoff-1992**
Fixes #110 the offset was not included when footers where not been used.

### CODE REVIEW

File: Sources/Shared/Core/VerticalBlueprintLayout.swift [hunk 1]
```diff
@@ -209,6 +209,8 @@
           layoutAttribute.min = headerAttribute?.frame.origin.y ?? nextY
           footerAttribute = layoutAttribute
           nextY = layoutAttribute.frame.maxY
+        } else {
+            nextY = nextY + sectionInset.bottom
```
Comment (zenangst) [inline Sources/Shared/Core/VerticalBlueprintLayout.swift]: Looks like this got a bit much indentation

**Comment by christoff-1992** [comment]
> Looks good, nice catch @christoff-1992

Ahh give us a second before merging!.. Il update the indentation, also realise that this probably needs to be the sectionMaxY not the nextY or it'l offset against the last cell.

**Comment by christoff-1992** [comment]
@zenangst - Thats better :D .. wish you could set indentation per workspace and not just as an overall preference!

**Comment by zenangst** [comment]
@christoff-1992 I think that should be possible if we set it on the project 🤔

**Comment by christoff-1992** [comment]
> @christoff-1992 I think that should be possible if we set it on the project 🤔

🤣 - I should have googled before saying that! .. looks like you can.

**Comment by zenangst** [comment]
@christoff-1992  No worries mate :)

**Pull request merged**